src/Service/PriceRuleService.php

```diff
@@ -426,4 +426,23 @@ private function duplicatePriceRuleTablesData($tables, $priceRuleTablesData, $ne
 
         return true;
     }
+
+    /**
+     * @param Cart $cart
+     *
+     * @return float|null
```

Comment:
It's never null, it's always int|float, change additionalShippingCosts to be a 0.00 to make it float